spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -358,12 +358,22 @@ def getRegistrationStackSh(saltEnabled):
     function getY_CLIENT_CODE_BASE() {{
         local BASE=""
         local VERSION=""
-        if [ -f /etc/redhat-release ]; then
-            grep -v '^#' /etc/redhat-release | grep -q '\(Red Hat\)' && BASE="res"
-            VERSION=`grep -v '^#' /etc/redhat-release | grep -Po '(?<=release )\d+'`
+        # SLES ES6 is a special case; it will install a symlink named
+        # centos-release pointing to redhat-release which will make the
+        # original test fail; reverting the checks does not help as this
+        # will break genuine CentOS systems. So use the poor man's approach
+        # to detect this special case. SLES ES7 does not have this issue
+        # https://bugzilla.suse.com/show_bug.cgi?id=1132576
+        # https://bugzilla.suse.com/show_bug.cgi?id=1152795
+        if [ -L /usr/share/doc/sles_es-release ]; then
```

Comment:
Does SLES ES7 has this file as well?

Reply:
No, they have the symlink in /usr/share instead (not in the doc directory). But SLES ES7 is not affected anyway as it does not contain the bogus /etc/centos-release link in the sles_es-release-server package. So SLES ES7 gets correctly detected as "res" as we expect.

Reply:
ok